src/fty_common_utf8.cc

```diff
@@ -329,6 +329,14 @@ s_jsonify_translation_string (const char *key, va_list args)
 {
     log_trace ("called");
 
+    // check if 'key' is already in JSON format
+    // since we may start/end with {{vari}} and we don't want to go through the whole string,
+    // check just first two and last two characters
+    size_t length = strlen (key);
+    if (key[0] == '{' && key[1] != '{' && key[length-2] != '}' && key[length-1] == '}') {
```

Comment:
just for the sanity check if length is at least 2

Reply:
good point, I missed ;)

Reply:
Good catch!